xml/System.Threading.Tasks/Task`1.xml

```diff
@@ -2185,9 +2185,9 @@ For operations that do not return a value, you use the <xref:System.Threading.Ta
         <Parameter Name="timeout" Type="System.TimeSpan" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="timeout">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="timeout">The timeout after which the <see cref="T:System.Threading.Tasks.Task" /> should be faulted with a <see cref="T:System.TimeoutException" /> if it hasn't otherwise completed.</param>
+        <summary>Gets a <see cref="T:System.Threading.Tasks.Task`1" /> that will complete when this <see cref="T:System.Threading.Tasks.Task`1" /> completes or when the specified timeout expires.</summary>
+        <returns>The <see cref="T:System.Threading.Tasks.Task`1" /> representing the asynchronous wait.  It may or may not be the same instance as the current instance.</returns>
```

Comment:
```suggestion
        <returns>The <see cref="T:System.Threading.Tasks.Task`1" /> representing the asynchronous wait. It may or may not be the same instance as the current instance.</returns>
```